recipe/meta.yaml

```
@@ -10,21 +10,20 @@ source:
   sha256: ed7d092c72fad9f6f839e1501229490cb278403ae95529a02860f3012f02bc0a
 
 build:
-  noarch: python
   number: 0
```

@psteyer conda-forge has the same version released for this package, could you please increase the build number for ours?

```suggestion
  number: 1
```

---

or maybe I remember wrong, @skupr-anaconda ?

---

The channel has higher priority. So there is no correlation between the package build number and the channel. If the package is available in the ❄️  channel then the solver chooses it and ignores the conda-forge. 
In short, for a new package, the build number should be set to 0 as our common practice.

---

Got it, thank you. I'll ask more info on slack.

---

Just to double check, we will be keeping it at `number: 0` then, since it's the first build for the channel?

---

yes